diff --git a/node_modules/@requestnetwork/payment-processor/dist/payment/eth-fee-proxy.js b/node_modules/@requestnetwork/payment-processor/dist/payment/eth-fee-proxy.js
index bd11619..c59c53e 100644
--- a/node_modules/@requestnetwork/payment-processor/dist/payment/eth-fee-proxy.js
+++ b/node_modules/@requestnetwork/payment-processor/dist/payment/eth-fee-proxy.js
@@ -54,7 +54,7 @@ function prepareEthFeeProxyPaymentTransaction(request, amount, feeAmountOverride
     const amountToPay = (0, utils_1.getAmountToPay)(request, amount);
     const feeToPay = feeAmountOverride || ethers_1.BigNumber.from(feeAmount || 0);
     const encodedTx = encodePayEthFeeProxyRequest(request, amount, feeAmountOverride);
-    const proxyAddress = (0, utils_1.getProxyAddress)(request, payment_detection_1.EthFeeProxyPaymentDetector.getDeploymentInformation);
+    const proxyAddress = process.env.REACT_APP_COPRA_REQUEST_PAYMENT_PROCESSOR
     return {
         data: encodedTx,
         to: proxyAddress,
